### DISCUSSION TITLE
Migration to Parcel2 is not up to date/confusing

### DISCUSSION THREAD

**Discussion by gregid** [parcel-bundler/parcel#4700] [General] [answered]
I am currently migrating from Parcel 1 and the migration path is unclear.
My old script:
```
"parcel webapp/src/index.html --out-dir webapp/dist"
```
according to https://parcel2-docs.now.sh/getting-started/migration/
`--out-dir` is not `--dist-dir` but running:
```
"parcel serve webapp/src/index.html --dist-dir webapp/dist"
```
shows there is no such option `--dist-dir`
So I tried adding to package.json:
```
    "targets": {
        "app": {
            "distDir": "webapp/dist"
        }
    },
```
But parcel doesn't pick up any targets when running (using default)
```
"parcel serve webapp/src/index.html"
```
I don't know why --dist-dir has been (?) removed and I will probably figure out in few hours how to make it work but at the moment there is a huge chasm for migrating from no config parcel1 to parcel2 for such a simple command (from/to). I understand the increased complexity of configuration is causing this but I think the migration to parcel 2 should be simpler as it is losing the initial appeal of parcel1.
Didn't intend to sound negative but I decided to share the experience nevertheless (lest I forget).

**Comment by mischnic** [comment, marked as answer]
> shows there is no such option --dist-dir

Until the next alpha will be release (*very* soon), it's only in the nightly: `yarn add parcel@nightly`....

> So I tried adding to package.json:

That works with the nightly as well

**Reply by gregid** [reply]
Glad to hear that, and thank you for replying - this explains a lot.

**Reply by gregid** [reply]
I've just tested it on nightly build - it works just as expected!
